demosplan/DemosPlanCoreBundle/Permissions/Permissions.php

```diff
@@ -526,7 +524,7 @@ static function (Permission $permission) {
     /**
      * Setze die Rechte, die ein Verfahren betreffen.
      */
-    protected function setProcedurePermissions(): void
+    public function setProcedurePermissions(): void
```

Comment:
leads to error in \demosplan\DemosPlanCoreBundle\Permissions\ProjectPermissions::setProcedurePermissions

Reply:
Thank you, unfortunately this is part of https://github.com/demos-europe/demosplan-core/pull/2991/files#diff-5e713faa2c7b521d13b127049eebd3f676f2ae97460924b7a75c9f100c033561 due to an error in my commits.

Reply:
Done in daa39454